www/pages/blog/release/v0-23-0.md

```diff
@@ -0,0 +1,120 @@
+              </tr>
+            `;
+          });
+        }
+      </table>
+    </body>
+  `
+}
+
+export { getBody }
+```
+
+You can then access `/artists/` and see the content!  💥
+
+
+![Server Side Rendering example](/assets/blog-images/ssr.webp)
+
+> _In the above screenshot, we can also see a demonstration of custom rendering using LitSSR with the `<simple-greeting>` component._
+
+## Interpolate Frontmatter
+At the risk of implementing / using a custom templating system (e.g. handlebars, nunjucks, etc), which is something Greenwood strives to avoid, but still recognizing that requiring JavaScript though our _graph.json_ for static sites can be a bit cumbersome.  With this new feature, which is a flag that can be enabled in your _greenwood.config.js_, frontmatter in your markdown will be available in your HTML or markdown.  Great for `<meta>` tags!
+
+### However it works
+So given the following frontmatter
+```md
+---
+template: 'post'
+title: 'Git Explorer'
+emoji: '💡'
+date: '04.07.2020'
+description: 'Local git repository viewer'
+image: '/assets/blog-post-images/git.png'
+---
+```
+
+And enabling the feature in _greenwood.config.js_
+```js
+export default {
+  interpolateFrontmatter: true
+}
+```
+
+You access the frontmatter data in the markdown or HTML on a _per page instance_ following the convention of JavaScript [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), and Greenwood will interpolate those values at build time.
+
+```md
+# My Blog Post
+
+<img src="${globalThis.page.image}" alt="Banner image for ${globalThis.page.description}">
+
+Lorum Ipsum.
+```
+
+```html
+<html>
+  <head>
+    <title>My Blog - ${globalThis.page.title}</title>
+    <meta property="og:title" content="My Blog">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://www.myblog.dev">
+    <meta property="og:image" content="https://www.myblog.dev/${globalThis.page.image}">
+    <meta property="og:description" content="My Blog - ${globalThis.page.description}">
+  </head>
+  <body>
+    <content-outlet></content-outlet>
+  </body>
+</html>
+```
+
+## Learn More
+
+To learn more about SSR and the full API please check out our docs on [SSR](/docs/server-rendering/) and [interpolateFrontmatter](/docs/config#interpolateFrontmatter).  For custom SSR, we have [plugin docs](/plugins/renderer/) and a [Lit Renderer plugin](https://github.com/ProjectEvergreen/greenwood/tree/master/packages/plugin-renderer-lit) you can start using.  As referenced at the start of the blog post, the SSR feature is brand new and we have many plans to incorporate future features and enhancments around [hydration, statically exporting content from server routes, and more](https://github.com/ProjectEvergreen/greenwood/issues?q=is%3Aissue+is%3Aopen+label%3Assr+label%3Apriority)!  Feedback appreciated and we cant wait to see what you end building!  🙏
```

Comment:
"enhancements"

"Feedback **is** appreciated and we can't wait to see what you build!